src/linker/runner.py

```diff
@@ -47,11 +51,12 @@ def main(
     snake_main(argv)
 
 
-def get_singularity_args(input_data: List[Path], results_dir: Path) -> str:
-    input_file_paths = ",".join(file.as_posix() for file in input_data)
+def get_singularity_args(config: Config, results_dir: Path) -> str:
+    input_file_paths = ",".join(file.as_posix() for file in config.input_data)
     singularity_args = "--no-home --containall"
-    LINKER_TEMP.mkdir(parents=True, exist_ok=True)
-    singularity_args += f" -B {LINKER_TEMP}:/tmp,{results_dir},{input_file_paths}"
+    linker_tmp_dir = LINKER_TEMP[config.computing_environment]
```

Comment:
Why can't think be `/tmp/linker` in both? Actually, I'd prefer it to be implementation-specific (e.g. `/tmp/linker/step_1_step_1_python_pandas/` in the long run to ensure isolation.

Reply:
subdirectories of `/tmp` won't exist on the compute node when `singularity exec` is run, and I was too lazy to try to figure out how to create it with a `--task-prolog`, but this is something we could try doing

Reply:
Ah I see. Makes sense, and not a big deal.